deployments/properties/azure-kubernetes-service-helm.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Deploy to AKS with Helm",
+    "description": "Deploy an application to an Azure Kubernetes Service Cluster using Azure Credentials and Helm",
```

Comment:
What is the significance of saying "using Azure Credentials" as it would be implicit. can that be removed?

Reply:
Removed